frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1266,6 +1494,14 @@ function ReaderFooter:onChangeScreenMode()
     self:resetLayout(true)
 end
 
+function ReaderFooter:onSetPageHorizMargins(h_margins)
+    if self.settings.progress_margin then
+        self.settings.progress_margin_width = math.floor((h_margins[1] + h_margins[2])/2)
```

Comment:
Oh, (following my previous comment) I'm just seeing that. So does that mean it's updated on any book load? :)